db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_os_types.rb

```diff
@@ -1,28 +1,46 @@
 #
 # Description: provide the dynamic list content from available operating systems
 #
-os_list = {'unknown' => '<Unknown>', 'linux' => 'Linux', 'windows' => 'Windows'}
-selected_os = 'unknown'
+class AvailableOsTypes
```

Comment:
@Fryguy @pkomanek @gmcculloug 
@Fryguy had a suggestion that the classes should be namespaced so that we don't have conflicts when testing. should we create the class name based on the path 
e.g. for this one ManageIQ::Cloud::Orchestration::Operations::AvailableOsTypes or
ManageIQ::Cloud::Orchestration::Operations::Methods::AvailableOsTypes.

@Fryguy please suggest.
Is the preferred way to have 
```
module ManageIQ
  module Cloud
       module Orchestration
           module Operations
                module Methods
                     class AvailableOsTypes
                     end
          ...
          end
```

or

```
module ManageIQ::Cloud::Orchestration::Operations::Methods
  class AvailableOsTypes
  end
end
```